documentation/airport-guides/rjss.md

```diff
@@ -0,0 +1,59 @@
+# RJSS - Sendai Airport
+
+## About this facility
+
+Sendai Airport (alternatively referred to as Sendai International Airport) is a Japanese airport in the city of Sendai, Miyagi Prefecture. 
+
+Opened in 1940 as a military airfield, despite the damage during World War II and the 2011 Tohoku earthquake and tsunami, it now connects to major cities in Japan and nearby Asia.
+
+## Positions
+
+|Callsign             | Frequency  | Remarks
+|---------------------|:----------:|:---------------------------------------------:|
+|"SENDAI GROUND"      | 121.700MHz |                                               |
+|"SENDAI TOWER"       | 118.700MHz | Primary Frequency                             |
+|                     | 126.200MHz |                                               |
+|"SENDAI DEPARTURE"   | 120.000MHz |                                               |
+|"SENDAI APPROACH"    | 118.700MHz |                                               |
+|"SENDAI RADAR"       | 121.200MHz |                                               |
+|"SENDAI TCA"         | 121.025MHz | For Terminal Control Area Advisory            |
+|"SENDAI ATIS"        | 126.450MHz |                                               |
+
+## Airspace
+
+There are four radar positions in the Sendai Approach Control Area (ACA) — Depature, Approach, Radar, and TCA.
+
+As the name suggests, the Departure and Approach positions are for departing and arriving aircrafts to Sendai Airport.
+
+The Radar position is used as a combined position of Depature and Approach.
+
+Finally, the TCA position provides advisory service to VFR traffics in the Sendai Terminal Control Area. 
+
+## Instrument Procedures
+###Departures
+
+Departing aircrafts will be assigned the Steed Three Depature, Derby Three Departure, or Cubic Three Departure; they are generally assigned to aircrafts departing southbound, westbound, and northbound, respectively. Pilots are given an initial altitude of FL160. 
+
+Due to obstructions, pilots departing RWY09 are required to climb at a 5.0% gradient up to 500FT, and those departing RWY27 are required to climb at a 5.0% gradient up to 1000FT, except for the Derby Three Depature, where pilots are required to climb at a 5.9% gradient until 1300FT.
+
+###Arrivals
+
+Tokyo Control will assign STARs to aircrafts and give initial descent instructions.
+
+After handed off from Tokyo Control, Sendai Approach is responsible for: 
+* issuing further descent clearance;
+* vectoring the aircraft to the final approach course if necessary;
+* issuing approach clearance
+* ensuring sufficient spacing between aircrafts;
+
+YUCCA is the Final Approach Fix (FAF) for ILS 27, meaning that aircrafts can be cleared for the approach after this fix. Approach should issue a descent clearance so that the aircraft is at 1500' at YUCCA.
```

Comment:
This won't work in the sim, since the minimum intercept altitude is 1,700ft. Might be worth mentioning it here.